content/blog/2019/04/2019-04-03-security-advisory.adoc

```diff
@@ -12,7 +12,7 @@ What's going on?
 
 The Jenkins security team triages https://jenkins.io/security/#reporting-vulnerabilities[incoming reports both to Jira and our non-public mailing list].
 Once we've determined it is a plugin not maintained by any Jenkins security team members, we try to inform the plugin maintainer about the issue, offering our help in developing, reviewing, and publishing any fixes.
-Sometimes the affected plugin is unmaintained, or maintainers don't respond to these notifications or the followup emails we send.
+Sometimes the affected plugin is unmaintained, or maintainers don't respond to these notifications or the followup emails we send in a timely manner.
```

Comment:
Minor phrasing improvement placing the modifier ("in a timely manner") nearer the thing it is modifying (the response).  Optional change, not mandatory.

```suggestion
Sometimes the affected plugin is unmaintained, or maintainers don't respond  in a timely manner to the notifications or the followup emails we send.
```